src/components/table/OcTableFiles.vue

```diff
@@ -630,16 +629,6 @@ export default {
             type: "folder",
             size: "5324435",
             mdate: "Mon, 11 Jan 2021 14:34:04 GMT"
-          },
```

Comment:
Why delete this example?

Reply:
If this was deleted in the gist I sent to you @paulcod3 it was not intentional. Please restore. ;-)